django_freeradius/base/models.py

```diff
@@ -819,3 +819,29 @@ def _remove_files(self):
         path = getattr(self, strategy_filemap.get(self.strategy)).path
         if os.path.isfile(path):
             os.remove(path)
+
+
+class AbstractRadiusToken(TimeStampedEditableModel, models.Model):
+    id = None
+    modified = None
```

Comment:
why are you removing the `id` and `modified` fields? Is there a specific reason?

Maybe in this case we can inherit from [UUIDModel](https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/base.py#L21):

```python
from openwisp_utils.base import UUIDModel


class AbstractRadiusToken(UUIDModel):
    # ...
```

Reply:
Because the key is itself a primary key and `id` will just be a redundant field. And I think modified field in the token model will hardly be of any use as the tokens are created and then deleted on authentication and admins will not be modifying the tokens manually, they will simply regenerate them if required.
@nemesisdesign

Reply:
ok, just add a comment in the code regarding this detail please